core/App/screens/ProofRequestAccept.tsx

```diff
@@ -45,6 +36,73 @@ const ProofRequestAccept: React.FC<ProofRequestAcceptProps> = ({ visible, proofI
       textAlign: 'center',
       marginTop: 30,
     },
+    delayMessageText: {
+      textAlign: 'center',
+      marginTop: 20,
+    },
+  })
+
+  switch (status) {
+    case ProofState.RequestReceived:
```

Comment:
The view must be integrated integrated on the screen from the Verifer side (`ProofRequesting`) not the Prover's

Reply:
done